justfile

```diff
@@ -0,0 +1,34 @@
+# useful to run when submitting a PR
+pr: fmt check clippy
+
+# format src and fuzzer
+fmt:
+    cargo fmt
+    cargo fmt --manifest-path fuzz/Cargo.toml
+
+# type-check with and without features, and the fuzzer
+check:
+    cargo check
+    cargo check --all-features
```

Comment:
Isn't it sufficient to just have `cargo check --all-features`?

Reply:
It is possible for the code to work with `--all-features` but not with default features, such as when a `use` statement required for default features was accidentally `#[cfg(...)]`'d to a non-default feature.

Reply:
Ah, that makes sense. In that case, perhaps add a comment explaining that so it's not removed in the future.